templates/todo/web/react-fluent/package-lock.json

```diff
@@ -13296,7 +13296,7 @@
       "integrity": "sha512-nRCcW9Sj7NuZwa2XvH9co8NPeXUBhZP7CRKJtU+cS6PW9FpCIFoI5ib0NT1ZrbNuPoRy0ylyCaUL8Gih4LSyFg==",
       "dev": true,
       "dependencies": {
-        "minimatch": "3.0.4"
+        "minimatch": "3.0.5"
```

Comment:
Similar changes not required for generators/repo/package-lock.json?

Reply:
In all other places, the dependecy is listed as `^3.0.4`, which makes `npm` to automatically look for the latest version.
There are only these 2 cases where the version is pinned directly.

Reply:
True - but the package-lock.json will force it to use the version stamped in it at time of last commit vs pulling latest version.